packages/@magic-sdk/react-native-bare/README.md

```diff
@@ -69,4 +71,44 @@ Please note that as of **v14.0.0** our React Native package offerings wrap the `
 We have also added an optional `backgroundColor` prop to the `Relayer` to fix issues with `SafeAreaView` showing the background. By default, the background will be white. If you have changed the background color as part of your [custom branding setup](https://magic.link/docs/authentication/features/login-ui#configuration), make sure to pass your custom background color to `magic.Relayer`:
 ```tsx
 <magic.Relayer backgroundColor="#0000FF"/>
+```
+
+## 🙌🏾 Troubleshooting
+
+### Symlinking in Monorepo w/ Metro
+
+For React Native projects living within a **monorepo** that run into the following `TypeError: Undefined is not an object` error:
+
+<img width="299" alt="Screenshot 2022-11-23 at 12 19 19 PM" src="https://user-images.githubusercontent.com/13407884/203641477-ec2e472e-86dc-4a22-b54a-eb694001617e.png">
+
+When attempting to import `Magic`, take note that the React Native metro bundler doesn’t work well with symlinks, which tend to be utilized by most package managers.
+
+For this issue consider using Microsoft's [rnx-kit](https://microsoft.github.io/rnx-kit/docs/guides/bundling) suite of tools that include a plugin for metro that fixes this symlink related error.
+
+### Handling internet connection problems
+
+When the app has internet connectivity issues, the relayer might behave in ways you don't expect. That is why it is important to use [@react-native-community/netinfo](https://www.npmjs.com/package/@react-native-community/netinfo) to track the internet connection state of the device, and refresh your UI when the connection is re-established. For your convenience, we've also added a hook that uses this library behind the scenes:
```

Comment:
Any feelings on calling out the new error message that will be returned here?

Reply:
Updated 👍